assignment4/islandCount.py

```diff
@@ -0,0 +1,57 @@
+# Author: Li Ling
+# Last modified: 2017-06-25
+
+class IslandCounter(object):
```

Comment:
@windo I looked a bit more at this point, and check Google's python style guide. In the guide, it says that any classes that don't inherit from another base class shall explicitly inherit from **object**. And yet as @marijja said, it doesn't really seem to make any difference for the code per se. So what's the guideline here? Is it a good practice to explicitly inherit from **object** even though Python 3 is used?